docs: document possibility to parameter scriptPattern (#389)
fastify/fastify-autoload: README.md

  ```

> - `scriptPattern` (optional) - Regex to override the script files accepted by default
>   ```js

Reviewer: ```suggestion
- `scriptPattern` (optional) - Regex to override the script files accepted by default. You should only use this option
with [customization hooks](https://nodejs.org/docs/latest/api/module.html#customization-hooks) 
provider, for example `ts-node`. Otherwise, widening the acceptance extension here will result in error.

  ```js
```

  fastify.register(autoLoad, {
    dir: path.join(__dirname, 'plugins'),
    scriptPattern: /(?<!\.d)\.(ts|tsx)$/
  })

> 

Reviewer 2: ````suggestion
```
````

  ```

- `scriptPattern` (optional) - Regex to override the script files accepted by default. You should only use this option

> with [customization hooks](https://nodejs.org/docs/latest/api/module.html#customization-hooks) 
> provider, for example `ts-node`. Otherwise, widening the acceptance extension here will result in error.

Reviewer 3: ```suggestion
with a [customization hooks](https://nodejs.org/docs/latest/api/module.html#customization-hooks) 
provider, such as `ts-node`. Otherwise, widening the acceptance extension here will result in error.
```
